torch/cuda/memory.py

```diff
@@ -461,6 +461,13 @@ def _format_count(cnt, pref_cnt):
             )
 
     lines.append("=" * 75)
+    lines.append("        Metric         | cudaFree   | cudaMalloc | cudaMalloc retries      ")
```

Comment:
It seems that all of these stats /should/ be able to follow the cur/peak/totalloc/totfreed pattern.